token-lending/README.md

```diff
@@ -1,15 +1,118 @@
-# Token-lending program
+# Token Lending program
 
 A lending protocol for the Token program on the Solana blockchain inspired by Aave and Compound.
 
 Full documentation is available at https://spl.solana.com/token-lending
 
 Web3 bindings are available in the `./js` directory.
 
-### On-Chain Programs
+### On-chain programs
+
+Please note that only the lending program deployed to devnet is currently operational.
 
 | Cluster | Program Address |
 | --- | --- |
 | Mainnet Beta | [`LendZqTs8gn5CTSJU1jWKhKuVpjJGom45nnwPb2AMTi`](https://explorer.solana.com/address/LendZqTs7gn5CTSJU1jWKhKuVpjJGom45nnwPb2AMTi) |
-| Testnet | [`LendZqTs8gn5CTSJU1jWKhKuVpjJGom45nnwPb2AMTi`](https://explorer.solana.com/address/LendZqTs8gn5CTSJU1jWKhKuVpjJGom45nnwPb2AMTi?cluster=testnet) |
-| Devnet | [`LendZqTs8gn5CTSJU1jWKhKuVpjJGom45nnwPb2AMTi`](https://explorer.solana.com/address/LendZqTs8gn5CTSJU1jWKhKuVpjJGom45nnwPb2AMTi?cluster=devnet) |
\ No newline at end of file
+| Testnet | [`6TvznH3B2e3p2mbhufNBpgSrLx6UkgvxtVQvopEZ2kuH`](https://explorer.solana.com/address/6TvznH3B2e3p2mbhufNBpgSrLx6UkgvxtVQvopEZ2kuH?cluster=testnet) |
+| Devnet | [`6TvznH3B2e3p2mbhufNBpgSrLx6UkgvxtVQvopEZ2kuH`](https://explorer.solana.com/address/6TvznH3B2e3p2mbhufNBpgSrLx6UkgvxtVQvopEZ2kuH?cluster=devnet) |
+
+### Deploy a lending program (optional)
+
+This is optional! You can simply add your own market and reserves to the existing on-chain programs.
+
+You can skip these steps and use the [Token Lending CLI](./cli/README.md) to create a lending market and add reserves to it.
```

Comment:
nit: what does "it" mean here? the already existing on-chain programs?

Reply:
Ah, `it` here is `a lending market`. But I can change this to make it more clear.

Reply:
Done.